### PR TITLE
update needed for arabic

### PR DESCRIPTION
**Pull request by avenirer**

### CODE REVIEW

**Comment by benedmunds** [comment]
Why the replacement of arabic with english in some cases?

**Comment by avenirer** [comment]
Oups... Did I do that? I see a mistake made to bulgarian, but I just corrected...

**Comment by avenirer** [comment]
Don't merge yet. I have to append to a lot of languages the new strings.

**Comment by benedmunds** [comment]
Ok, just @ mention me when its ready

**Comment by avenirer** [comment]
Finished... Hope it looks ok. If not, I can make it prettier with tabs

File: language/arabic/ion_auth_lang.php [hunk 1]
```diff
@@ -49,7 +49,25 @@
...
+$lang['email_activate_subheading'] = 'Please click this link to %s.';
+$lang['email_activate_link']       = 'Activate Your Account';
+// Forgot Password Email
 $lang['email_forgotten_password_subject']    = 'Forgotten Password Verification';
+$lang['email_forgot_password_heading']    = 'Reset Password for %s';
+$lang['email_forgot_password_subheading'] = 'Please click this link to %s.';
+$lang['email_forgot_password_link']       = 'Reset Your Password';
+// New Password Email
 $lang['email_new_password_subject']          = 'New Password';
-$lang['email_activation_subject']            = 'Account Activation';
```
Comment (benedmunds) [inline language/arabic/ion_auth_lang.php]: Why are you removing this?

**Comment by benedmunds** [comment]
@avenirer added a comment

**Comment by avenirer** [comment]
@benedmunds I actually didn't removed them, I just moved them so that they resemble to the english file structure (same line is upper - line 61)

**Comment by benedmunds** [comment]
Ah thanks! My bad.  This looks good!

**Pull request merged**